app/controllers/comments_controller.rb

```diff
@@ -1,11 +1,10 @@
 class CommentsController < ApplicationController
-  def create
+  def build
```

Comment:
This method still wants to be `create`.

You'll want to test this stuff still works after your changes, rather than assuming they do.  At this point you might as well write tests for comments, or remove all of this stuff until you are required to implement them as part of the curriculum, up to you how you want to approach it.

Reply:
Hi Kris, I'm confused now.  You said this should be new or build rather than create? I think I will strip out the comments stuff until later but could you clarify first?  Thanks.